### PR TITLE
CR-8947-logs

### PR DESCRIPTION
**Pull request by saffi-codefresh**

### CODE REVIEW

File: codefresh.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ steps:
 
   test_with_server:
     title: Test unit-tests
-    image: node:10.13-alpine
+    image: node:10.24-alpine
```
Comment (ziv-codefresh) [inline codefresh.yml:13]: why are you using such old node?

File: ssrf-allow.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const getAllowList = () => {
+    if ('EXTERNAL_YAML_URL_ALLOW_LIST' in process.env) {
```
Comment (ziv-codefresh) [inline ssrf-allow.js:2]: simplify it

File: package.json [hunk 3]
```diff
@@ -1,17 +1,23 @@
 {
   "name": "@codefresh-io/ssrf-safe",
-  "version": "0.0.10",
+  "version": "0.1.0-rc1",
```
Comment (ziv-codefresh) [inline package.json:3]: version

**Review by ziv-codefresh — APPROVED** [review]

File: codefresh.yml (hunk 1 above)
Comment (saffi-codefresh) [reply codefresh.yml:13]: That what we use with cf-api. test same.

**Pull request merged**